resources/skins/nudus/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<skin version="0.01">
+  <author>
+    <name>anonymous</name>
+    <email>anonymous</email>
+  </author>
+  <palette>
+    <color key="1">#4861F0</color>
```

Comment:
Keys are taken from [here](https://github.com/martinrotter/rssguard/blob/master/src/librssguard/miscellaneous/skinfactory.h#L15).

Reply:
Yes, but the colours are already changed. I only left the first one the same as in vergilius.